doc/source/index.rst

```diff
@@ -67,6 +67,7 @@ About
    people
    organizations
    publications
+   blog-post
```

Comment:
The matching file is missing from this PR

Reply:
push file